sarahricker/thinkery
# [THNK-48] Blog styles

## JIRA Ticket

[THNK-48](https://wpengine.atlassian.net/browse/THNK-48)

## Description

A few sentences describing the overall goals of the pull request's commits.

Sets up Blog templates + block areas post types to control widgets + archive headers. 

## Impacted Areas in Application

List general components of the site that this PR will affect:

* shortcodes, 
* block areas
* Blog


## Deploy Notes

Notes regarding deployment the contained body of work.
If you need to change something in WordPress after deploying, make the necessary steps crystal clear.

- Set up Block area widgets w/ the following import file
- Visit Customizer + turn on block area widget for blog archive page

## Steps to Test or Reproduce

1. pull down branch
2. follow deploy notes
3. see blog headers apply w/ dynamic post titles + blog layout that matches design

## Random GIF

Include one random GIF to "pay" your reviewers in humor.

![Thanks for reviewing my code!](https://media.giphy.com/media/j2HWP75PB3wd4GM7o3/giphy-downsized.gif)
